pom.xml

```diff
@@ -18,6 +18,9 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
```

Comment:
findbugs-maven-plugin v3.0.1 is unusable in a French OS:
https://github.com/gleclaire/findbugs-maven-plugin/issues/18